### PR TITLE
ui: add `.dockerignore`

### PR DESCRIPTION
**Pull request by NicolasT**
Forgot to add this as part of 46f0a1699b8da2352308a3b64ab40de9d0188cbd.

See: 46f0a1699b8da2352308a3b64ab40de9d0188cbd

### CODE REVIEW

File: ui/.dockerignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules/
+build/
+tmp/
```
Comment (carlito-scality) [inline ui/.dockerignore:4]: I think ```tmp```is not necessary here.

Comment (carlito-scality) [reply ui/.dockerignore]: We can add ```junit/``` to this list too

Comment (NicolasT) [reply ui/.dockerignore:4]: Cypress created `tmp/cypress/screenshots/...` on my system.

Comment (carlito-scality) [reply ui/.dockerignore:4]: oh yes. it is for debug

**Review by carlito-scality — APPROVED** [review]

**Pull request merged**